packages/contracts/contracts/L1/messaging/L1StandardBridge.sol

```diff
@@ -38,26 +39,16 @@ contract L1StandardBridge is IL1StandardBridge, CrossDomainEnabled {
      ***************/
 
     // This contract lives behind a proxy, so the constructor parameters will go unused.
-    constructor()
-        CrossDomainEnabled(address(0))
-    {}
-
-    /******************
-     * Initialization *
-     ******************/
-
-    /**
+        /**
      * @param _l1messenger L1 Messenger address being used for cross-chain communications.
      * @param _l2TokenBridge L2 standard bridge address.
      */
-    function initialize(
+    constructor(
```

Comment:
I believe the reason we had `initialize` here is to simplify to deployment process, which currently goes: 
1. Deploy the L1 Bridge
2. Deploy the L2 Bridge, get its address
3. Init the L1 Bridge with the L2 Bridge's address

That's likely why the integration tests are failing with `curl: (6) Could not resolve host: deployer`

Reply:
However... since the L2 Bridge is already deployed, I think we could get away with making this change now. :)

Reply:
Another issue here is that this sits behind a proxy, so the constructor never gets triggered.

Reply:
@smartcontracts Immutables are not in storage though and are inserted into the runtime code at deploy time which is why using the constructor will work with the proxy.

Reply:
@maurelian I was even thinking of moving the L2bridge address to be read from the predeploys lib. What do you think?

Reply:
> moving the L2bridge address to be read from the predeploys lib

Sorry, I'm not entirely sure what that would look like